projects/sandbox/train/train/train.py

```diff
@@ -44,6 +46,8 @@ def main(
     highpass: float,
     # augmentation args
     waveform_prob: float,
+    glitch_prob: float,
```

Comment:
For consistency, I think we should rename this to `glitch_frac`, as we now have a guaranteed number of glitches per batch